cursorless-talon/.gitignore

```
@@ -1,3 +1,4 @@
 *.flac
 data/
 .vscode/settings.json
+*.DS_Store
```

```suggestion
.DS_Store
```

I think this will work as well, no? The most common gitignores I've seen just use `.DS_Store` since we only need to match the literal name of the file.

---

Ah, I guess this comes from here: https://github.com/cursorless-dev/cursorless/blob/main/.gitignore#L8

We should probably fix it there as well.

---

Agreed that it should be fixed. I copied what was there in the existing `.gitignore` but prefer without the unnecessary wildcard. Personally I tend to generate `.gitignore` files using something like [gitignore.io](https://www.toptal.com/developers/gitignore/) which cover files you might not have thought of, although sometimes too many. In any case this suggests `.DS_Store` for `macOS`.

---

Do you want to just make that change as part of this PR? If so we can stamp and merge this one pretty quickly. If you're not comfortable I can do that as a separate change, but I don't think it's contentious.

---

Sorry. Done. Force pushed the update.